portality/templates/includes/header.html

```diff
@@ -11,11 +11,10 @@ <h2 class="sr-only">The Directory of Open Access Journals</h2>
                     {% include "includes/svg/logo.svg" %}
                 </a>
             </div>
-            <nav class="col-xs-8 col-sm-6">
+            <nav class="col-xs-8 col-sm-6" aria-label="Main actions">
```

Comment:
The aria label is repeated here

Reply:
Yes, that is intended. Again: in line with the W3 pattern.

Reply:
I mean that there is already a visually-hidden heading (`<h2 class="sr-only">Main actions</h2>`) which has the same function as an `aria-label` -- we don’t need both. The `sr-only` class is meant to include textual content as a guide for screen readers only.